packages/fxa-event-broker/src/queueworker/queueworker.service.ts

```diff
@@ -357,10 +339,6 @@ export class QueueworkerService
         await this.handleMessageFanout(message, 'password');
         break;
       }
-      case dto.APPLE_USER_MIGRATION_EVENT: {
```

Comment:
Should `APPLE_USER_MIGRATION_EVENT` also be removed from pubsub-proxy.controller.ts?

Reply:
I ended up deleting a few more things related to the Apple migration.